spectests/structs_encoding.go

```diff
@@ -101,7 +101,7 @@ func (a *AggregateAndProof) HashTreeRoot() ([32]byte, error) {
 }
 
 // HashTreeRootWith ssz hashes the AggregateAndProof object with a hasher
-func (a *AggregateAndProof) HashTreeRootWith(hh ssz.HashWalker) (err error) {
+func (a *AggregateAndProof) HashTreeRootWith(hh *ssz.Hasher) (err error) {
```

Comment:
This is a generated file so I am not really looking much into these extensive changes